src/controllers/trip.controller.js

```diff
@@ -116,6 +116,26 @@ class TripController {
     ResponseService.setSuccess(200, 'List of all available locations', availableLocations);
     return ResponseService.send(res);
   }
+
+  /**
+ *
+ *
+ * @static
+ * @param {req} req
+ * @param {res} res
+ * @returns {response} @memberof Trips
+ */
+  static async editOpenTripRequest(req, res) {
+    const { userData } = req;
+    const userId = userData.id;
+    const { tripId } = req.params;
+    const [, [{ dataValues }]] = await TripService.updateTrip(
```

Comment:
Where are you checking that the request is `open` before making the update?

Reply:
@adesege  I am checking it from [Here](https://github.com/andela/the_spinners-backend/blob/77f6cc98a02eee79959bced22c7d046b978f4ed5/src/middlewares/request.middleware.js#L10)

Reply:
Rename the name of the middleware to reflect the `status` you are checking for

Reply:
Yes @adesege  I finish to rename the middleware to what it does. You can have a look on the PR. thank you